### ISSUE TITLE
Add curl examples

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/specberus#826]
Would be really nice if https://github.com/w3c/specberus/blob/master/README.md#5-rest-api had some curl examples. As it's maybe once of twice a year that people need to use Specberus, it would be nice to have quick copy/paste curl examples that one can just use.

**Comment by deniak** [comment]
I've updated the examples section: https://github.com/w3c/specberus/blob/master/README.md#examples
Is it better now?

**Comment by marcoscaceres** [comment]
Much, though the reason I suggested curl was primarily to show how to directly POST local files. The problem with both ReSpec and BikeShed is that we need to generate files locally before putting them on a server somewhere... in general, we can't put the generated files on GitHub because gh-pages gets used to either:

 * serve the editors draft. 
 * only contain the source file - not the generated static file. 

The ideal workflow from the command line would be one where I can generate the static file using `respec2html`, and then use `curl` to POST that file directly to Specberus without having to deal with gh-pages.... Or even better: teaching Specberus about ReSpec/BS documents and having it automatically handle creating the static sources and spitting back the validation results.

**Comment by deniak** [comment]
Unfortunately, the API only works with [GET queries](https://github.com/w3c/specberus/blob/master/lib/api.js#L149) for now so you need to have your document uploaded somewhere if you want to use it.
I remember there was an attempt to add support for POST directly but some rules will not work as we rely on external services, such as the validators, so this will require some work before we can actually check a document just by POSTing it.
